folder/README.md

```diff
@@ -10,4 +10,11 @@ We are about to make a commit without best practices.
 
 I add new lines in order to see the differences...
 
+<<<<<<< Updated upstream
+=======
```

Comment:
I think that this equal, "=", symbols are useless. Please review this part in order to continue with the following task.